obspy/taup/data/1066b.nd

```diff
@@ -160,4 +160,4 @@ inner-core
  6257.10      11.277     3.549    13.064  183300.0    3666.0
  6295.10      11.279     3.550    13.069  183300.0    3666.0
  6333.00      11.280     3.550    13.074  183300.0    3666.0
- 6371.00      11.281     3.550    13.077  183300.0    3666.0
+ 6370.98      11.281     3.550    13.077  183300.0    3666.0
```

Comment:
Yea it was wrong - that's how I initially realized the error. I originally "fixed" the model as 200 meters just don't matter for the travel times. Now it is correct though.